src/static/css/infusion-docs.css

```diff
@@ -197,7 +202,8 @@ a:focus {
 }
 
 .infusion-docs-TOC li {
-    margin-bottom: 1rem;
+    padding: 0.6rem 0;
+    border-top: 0.1rem solid #efefef;
```

Comment:
Very subtle point: I wonder if this might work better as a border-bottom. As it is, a top border seems to create a slight separation between the first item in a section and the section header, and a lack of such separation between the last item and the next header, and I wonder if the reverse might come across better.